matrimony_admin/views.py

```
@@ -1,30 +1,33 @@
 from django.contrib.auth import authenticate, login, logout
-from django.views.generic import FormView
-from django.urls import reverse_lazy
-from .forms import AdminLoginForm, AdminProfileForm,NotificationDetailsForm
-from .models import BlockedUserInfo
-from U_auth.permissions import *
+from django.core.exceptions import ValidationError
+from django.core.validators import validate_email
 
-from U_auth.models import costume_user
+# Django ORM and database functions
 from django.db.models import Sum, Count, Q
```

Ruff

`django.db.models.Count` imported but unused

Remove unused import

(F401)